webhook/aws/sns_handler_test.go

```diff
@@ -379,6 +386,25 @@ func TestNotificationHandleError_ValidationErr(t *testing.T) {
 	mv.AssertExpectations(t)
 }
 
+func TestPublishClientTimeout(t *testing.T) {
+	fmt.Println("\n\nTestNotificationHandleError_ValidationErr")
+
+	ss, m, _, _ := SetUpTestSNSServerWithChannelSize(t, 1)
+	ss.channelClientTimeout = 1
+
+	// should not panic
+	testPublish(t, m, ss)
+
+	defer func() {
```

Comment:
Consider using the `stretchr/testify` library, with has an `assert.Panics` assertion that will clean up this code considerabliy.